helm/external-dns-app/templates/secret.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: "{{ .Values.name }}-route53-credentials"
-  namespace: "{{ .Values.namespace }}"
+  name: "{{ .Release.Name }}-route53-credentials"
+  namespace: "{{ .Release.Namespace }}"
   labels:
     {{- include "labels.common" . | nindent 4 }}
 data:
-  aws_access_key_id: {{ .Values.externalDNS.aws_access_key_id | b64enc }}
-  aws_secret_access_key: {{ .Values.externalDNS.aws_secret_access_key | b64enc }}
+  aws_access_key_id: {{ .Values.aws.credentials.awsAccessKeyID | b64enc }}
+  aws_secret_access_key: {{ .Values.aws.credentials.awsAccessSecretKey | b64enc }}
```

Comment:
renamed these and documented them via the values file (although they should be provided as a secret which will be merged into the values file)